api/src/infrastructure/kubernetes/payloads.rs

```diff
@@ -163,8 +162,8 @@ pub fn deployment_payload(
         BTreeMap::from([(IMAGE_LABEL.to_string(), strategy.image().to_string())])
     };
 
-    let volume_mounts = strategy.volumes().map(|volumes| {
-        let parent_paths = volumes
+    let file_mounts = strategy.files().map(|files| {
```

Comment:
Actually, I would say that `volume_mounts` is the correct name here because it maps to a Kubernetes primitive. 

```suggestion
    let volume_mounts = strategy.files().map(|files| {
``

Reply:
Followed the K8s primitives and reverted to previous version.